src/components/text/textTypes.js

```diff
@@ -0,0 +1,75 @@
+// @flow strict
+export type TextTypeType =
+  |'span'
```

Comment:
space? :D

Reply:
wow, i thought eslint will caught it so i didn't look carefully at indentation :D

Reply:
fixed